modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm

```diff
@@ -98,98 +100,121 @@ GLOBAL_LIST_EMPTY(customizable_races)
+							if(HAIR)
+								if(hair_color == "mutcolor")
+									accessory_overlay.color = owner.dna.features["mcolor"]
+								else if(hair_color == "fixedmutcolor")
+									accessory_overlay.color = fixed_mut_color
+								else
+									accessory_overlay.color = owner.hair_color
+							if(FACEHAIR)
+								accessory_overlay.color = owner.facial_hair_color
+							if(EYECOLOR)
+								accessory_overlay.color = owner.eye_color_left
 				else
-					switch(bodypart_accessory.color_src)
-						if(USE_ONE_COLOR)
-							accessory_overlay.color = mutant_bodyparts[key][MUTANT_INDEX_COLOR_LIST][1]
-						if(USE_MATRIXED_COLORS)
-							var/list/color_list = mutant_bodyparts[key][MUTANT_INDEX_COLOR_LIST]
-							for (var/n in color_layer_list)
-								var/num = text2num(n)
-								var/mutable_appearance/matrixed_acce = mutable_appearance(icon_to_use, layer = -layer)
-								matrixed_acce.icon_state = "[render_state]_[layertext]_[color_layer_list[n]]"
-								matrixed_acce.color = color_list[num]
-								matrixed_acce.alpha = specific_alpha
-								if (bodypart_accessory.center)
-									matrixed_acce = center_image(matrixed_acce, x_shift, bodypart_accessory.dimension_y)
-								accessories += matrixed_acce
-								if (mutant_bodyparts[key][MUTANT_INDEX_EMISSIVE_LIST] && mutant_bodyparts[key][MUTANT_INDEX_EMISSIVE_LIST][num])
-									var/mutable_appearance/emissive_overlay = emissive_appearance_copy(matrixed_acce, owner)
-									//if (bodypart_accessory.center)
-									//	emissive_overlay = center_image(emissive_overlay, x_shift, bodypart_accessory.dimension_y)
-									accessories += emissive_overlay
-						if(MUTCOLORS)
-							if(fixed_mut_color)
-								accessory_overlay.color = fixed_mut_color
-							else
-								accessory_overlay.color = owner.dna.features["mcolor"]
-						if(HAIR)
-							if(hair_color == "mutcolor")
-								accessory_overlay.color = owner.dna.features["mcolor"]
-							else if(hair_color == "fixedmutcolor")
-								accessory_overlay.color = fixed_mut_color
-							else
-								accessory_overlay.color = owner.hair_color
-						if(FACEHAIR)
-							accessory_overlay.color = owner.facial_hair_color
-						if(EYECOLOR)
-							accessory_overlay.color = owner.eye_color_left
-			else
-				accessory_overlay.color = override_color
-			if (accessories)
-				for (var/acces in accessories)
-					standing += acces
-			else
-				standing += accessory_overlay
-				if (mutant_bodyparts[key][MUTANT_INDEX_EMISSIVE_LIST] && mutant_bodyparts[key][MUTANT_INDEX_EMISSIVE_LIST][1])
-					var/mutable_appearance/emissive_overlay = emissive_appearance_copy(accessory_overlay, owner)
-					//if (bodypart_accessory.center)
-					//	emissive_overlay = center_image(emissive_overlay, x_shift, bodypart_accessory.dimension_y)
-					standing += emissive_overlay
-
-			if(bodypart_accessory.hasinner)
+					accessory_overlay.color = override_color
+
+				if(accessories)
+					for (var/acces in accessories)
```

Comment:
could just do `standing += accessories`, pretty sure

Reply:
You are absolutely correct.